components/ui/CustomLink.jsx

```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+
+export default function CustomLink({ link , text , style }) {
+    return (
+        <Link href={`${link ? link : "/"}`}>
+            <span className={`font-normal group transition-all duration-300 text-xl leading-7 relative inline-flex items-center hover:underline  ${style}`}>
```

Comment:
What does this class `group` do? is it from Tailwind or a custom?

Reply:
It helps us to target the parent for the hover effect on the children